R/read_csv.R

```diff
@@ -214,7 +214,8 @@ read_cmdstan_csv <- function(files,
       altrep = FALSE,
       progress = FALSE,
       skip = metadata$lines_to_skip,
-      col_select = col_select
+      col_select = col_select,
+      num_threads = 1
```

Comment:
The macOS tests were still crashing, this fixes that. Will work on removing vroom in favor of something else, so this is temporary.